cmd/manager/main_test.go

```diff
@@ -0,0 +1,16 @@
+package main_test
+
+import (
+	"testing"
+
+	main "github.com/codeready-toolchain/host-operator/cmd/manager"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestVars(t *testing.T) {
+	// simply verify that the vars exist.
+	// They will be populated by the `-ldflags="-X ..."` at build time
+	assert.Equal(t, "unknown", main.Commit)
+	assert.Equal(t, "unknown", main.BuildTime)
```

Comment:
Not getting point of this test.  What's the objective and what it test.  What I see here is, is just asserting the variable are initialized with particular value and nothing else ??

Reply:
@nurali-techie yes, this ensures that the variables exist. We can't do much at this level (unit testing), but in the future, we could have e2e tests to check the actual values (matching a commit hash and a datetime). Does this clarify?

Reply:
Test for variable exists, doesn't make much sense for statically defined variable.  If these variable are not exists then there test will failed with compile error.

Reply:
Sure, but this test is a tiny extra harness. 
We're not going to debate on whether we should keep or remove this test, are we?

Reply:
As Xavier said the point is to make sure we have main.Commit and main.BuildTime available. We didn't before (and it was a bug). So, the test could be just as simple as:
```
var _ = main.Commit
var _ = main.BuildTime
```
But `assert.Equal(t, "unknown", main.Commit)` would work fine too.